test/specs/repo/extensions/universal-language-selector.ts

```diff
@@ -5,9 +5,23 @@ describe( 'UniversalLanguageSelector', function () {
 
 	it( 'Should be able to see the language selector menu', async function () {
 		await browser.url( testEnv.vars.WIKIBASE_URL );
-		await $( '#searchInput' ).click();
-		await $( '.imeselector' ).click();
 
-		await expect( $( '.imeselector-menu h3' ) ).toHaveText( 'English' );
+		await $( '#searchform input' ).click();
+
+		// work around lang selector not showing up the first time
+		// blur the search bar
+		await $( '.page-Main_Page' ).click();
+		// focus search bar again, lang selector should be there now
+		await $( '#searchform input' ).click();
+
+		await $$( '.imeselector' )
+			.filter( async ( selector ) => selector.isClickable() )[ 0 ]
+			.click();
+
+		// we need to use getHTML, getText() is the empty string for some reason,
+		// does it think the element is hidden?
```

Comment:
https://webdriver.io/docs/api/element/getText/
If an element isn't *interactable* it returns empty, which seems frankly ridiculous to me

Reply:
Thanks for clarification, added a comment about that.